src/components/QueryForm.tsx

```diff
@@ -235,21 +168,25 @@ function QueryForm({
           </div>
         </div>
       </div>
-      <div className={isFederationAPI ? 'col-span-2 row-start-5' : 'col-span-2 row-start-4'}>
+      <div className={isFederationAPI ? 'col-span-2 row-start-6' : 'col-span-2 row-start-5'}>
         <ContinuousField
+          error={minNumSessionsError}
+          helperText={minNumSessionsHelperText}
           label="Minimum number of sessions"
-          onFieldChange={(label, value) => updateContinuousQueryParams(label, value)}
+          onFieldChange={(label, value) =>
+            validate(label, value, setMinNumSessionsError, setMinNumSessionsHelperText, 0)
```

Comment:
as mentioned above, I think we can just do the same here as we do for categorical variables, i.e. just pass the `updateContinuousOptions` handler (or what it is called) and handle the validation in plain JS based on the props